### PR TITLE
Containerized Docker Application Lifecycle - 01.2

### PR DESCRIPTION
**Pull request by mvelosop**
## Summary

 Guide <-> e-book syncing - Chapter 1 - Article 2

### CODE REVIEW

**Comment by mvelosop** [comment]
@mairaw let's start with chapters 1 & 2 only, just to check if the review process flows as expected.

File: docs/standard/containerized-lifecycle-architecture/what-is-docker.md [hunk 1]
```diff
@@ -3,47 +3,75 @@ title: What is Docker?
...
+Developers can use development environments on Windows, Linux, or macOS. On the development computer, the developer runs a Docker host where Docker images are deployed, including the app and its dependencies. Developers who work on Linux or on the Mac use a Docker host that is Linux-based, and they can create images only for Linux containers. (Developers working on the Mac can edit code or run the Docker command-line interface (CLI) from macOS, but as of this writing, containers don't run directly on macOS.) Developers who work on Windows can create images for either Linux or Windows Containers.
```
Comment (mairaw) [inline docs/standard/containerized-lifecycle-architecture/what-is-docker.md]:
```suggestion
Developers can use development environments on Windows, Linux, or macOS. On the development computer, the developer runs a Docker host where Docker images are deployed, including the app and its dependencies. Developers who work on Linux or on the Mac, use a Docker host that is Linux-based, and they can only create images for Linux containers. (Developers working on the Mac can edit code or run the Docker command-line interface (CLI) from macOS, but as of this writing, containers don't run directly on macOS.) Developers who work on Windows can create images for either Linux or Windows Containers.
```

File: docs/standard/containerized-lifecycle-architecture/what-is-docker.md [hunk 2]
```diff
@@ -3,47 +3,75 @@ title: What is Docker?
...
 
 To host containers in development environments and provide additional developer tools, Docker ships [Docker Community Edition (CE)](https://www.docker.com/community-edition) for Windows or for macOS. These products install the necessary VM (the Docker host) to host the containers. Docker also makes available [Docker Enterprise Edition (EE)](https://www.docker.com/enterprise-edition), which is designed for enterprise development and is used by IT teams who build, ship, and run large business-critical applications in production.
 
-To run [Windows Containers](/virtualization/windowscontainers/about/), there are two types of runtimes:
+To run [Windows Containers](https://msdn.microsoft.com/virtualization/windowscontainers/about/about_overview), there are two types of runtimes:
```
Comment (mairaw) [inline docs/standard/containerized-lifecycle-architecture/what-is-docker.md]: The previous link was correct. Please don't bring back msdn links. We're trying to get rid of them with this massive effort #2037.
On the e-book, you can use the full path: https://docs.microsoft.com/virtualization/windowscontainers/about/index

```suggestion
To run [Windows Containers](/virtualization/windowscontainers/about/), there are two types of runtimes:
```

**Review by mairaw — APPROVED** [review]
LGTM but left a couple of nits for you to consider plus a msdn link update that shouldn't be merged. Feel free to merge when the PR is ready.

File: docs/standard/containerized-lifecycle-architecture/what-is-docker.md (hunk 2 above)
Comment (mvelosop) [reply docs/standard/containerized-lifecycle-architecture/what-is-docker.md]: Ok, there were time I wasn't clear about the links and other where the just slipped 😔

**Pull request merged**